.github/CODEOWNERS

```diff
@@ -463,6 +463,9 @@
 # ServiceLabel: %DevOps
 # ServiceOwners:                                                   @narula0781 @ashishonce @romil07
 
+# PRLabel: %DevOpsInfrastructure
```

Comment:
As this is a management library, please move this to the management section around L942 and copy the format there to restrict the path pattern to the resource manager area.   Please update the label to be `DevOps - Infrastructure`, which I've created and added to the cross-repository common set.

Please be sure to maintain alphabetical ordering by tag.